### DISCUSSION TITLE
Understand Survey creation flow modification

### DISCUSSION THREAD

**Discussion by kiranharidas187** [Sunbird-inQuiry/Community#12] [Q&A] [answered]
@pallakartheekreddy @Prerna789 @manojSRawat We are currently working on Observation and Survey creation in the Sunbird creation part. While going through the Survey flow we have identified some modifications done in the flow ie when clicking on the Survey tile,  it takes us to a different page(PFA). As @Prerna789 and the team were working on these parts, please help us with the following points

- Is it a necessary change or can we revert it? 
- What changes have to be made to revert that change?

Attaching the screenshots for reference.

![Screenshot from 2022-05-19 11-16-09](https://user-images.githubusercontent.com/21116904/170635296-1e2589ab-11a2-4895-b20c-c55f71aeaf4d.png)
![Screenshot from 2022-05-19 11-16-12](https://user-images.githubusercontent.com/21116904/170635310-e4138c45-c5d4-491d-9f99-49b31d05a466.png)

**Comment by Krishnaj20** [comment, marked as answer]
@kiranharidas187 - You are referring to enabling survey contribution in the coKreat portal. Please initiate this discussion in the [coKreat](https://github.com/Sunbird-coKreat/Community/discussions) building block forums.
